.github/workflows/trigger-benchmark.yml

```diff
@@ -0,0 +1,34 @@
+name: Check if needs trigger CircleCI benchmark
+
+on:
+  pull_request:
+   types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  haslabel:
+    name: analyse labels
+    runs-on: ubuntu-latest
+    outputs:
+      benchmark: ${{ steps.haslabel.outputs.labeled-run-benchmark }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Labeled with run-benchmark
+        id: haslabel
+        uses: DanielTamkin/HasLabel@v1.0.4
```

Comment:
Thanks for this. Both VecSim + module-tests do this by hand. Cool to see an action for this.

Reply:
:+1: